apiary.apib

```diff
@@ -2377,7 +2377,7 @@ Load tables from Storage into a Workspace using zero-copy cloning functionality
 support any filtering or processing parameters. Only tables stored in Snowflake are supported.
 
 The cloned table will contain all columns of the source table (all of them as `VARCHAR(1048576)`)
-and a `_timestamp` system column. 
+and a `_timestamp` system column.
```

Comment:
Nějak tu jinou změnu kromě této nevidím kde by se přidával ten format.

Reply:
@Halama promiň, nejspíš jsem commitnul něco špatně :-1: Našel jsem to ve svým apiary hotový, tak jsem to dal do novýho commitu ;) Je to pushnutý